docs/windows.md

```diff
@@ -3,8 +3,8 @@
 ## Development Environment
 
 To setup a development environment in Windows:
-1) Install Git for Windows from https://gitforwindows.org
-1) Go to http://www.msys2.org/ and download the x86_64 installer
+1) Install [Git for Windows](https://gitforwindows.org)
```

Comment:
No, unfortunately this is a combination for errors. Msys uses posix style paths since it is a posix emulation layer, and mingw uses Windows style paths because it directly compiles programs for Windows using gcc.

For example, pre-commit uses git to get paths, and if we have msys git installed it will cause FileNotFoundErrors.

Git-for-Windows is compiled with Mingw-w64, unfortunately the two projects decided that there is too many changes to upstream them back to mingw-w64. So, there is no mingw-git in mingw-w64, and if you want one you need to install git-for-windows. There is some history on this here:
https://github.com/git-for-windows/git/issues/284

I think we have the CI setup with msys-git, but we probably don't want it like that.